CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to
+agree to a Contributor License Agreement (CLA) declaring that you have the right to,
+and actually do, grant us the rights to use your contribution. For details, visit
+https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need
+to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the
+instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/)
+or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+### Following our coding conventions
+
+We format all our code using the coding conventions in the
+[citus_indent](https://github.com/citusdata/tools/tree/develop/uncrustify)
+tool. This tool uses uncrustify under the hood.
+
+```bash
+# Uncrustify changes the way it formats code every release a bit. To make sure
+# everyone formats consistently we use version 0.68.1:
+curl -L https://github.com/uncrustify/uncrustify/archive/uncrustify-0.68.1.tar.gz | tar xz
+cd uncrustify-uncrustify-0.68.1/
+mkdir build
+cd build
+cmake ..
+make -j5
+sudo make install
+cd ../..
+
+git clone https://github.com/citusdata/tools.git
+cd tools/uncrustify
+make install
+```
+
+After installing like this you can run the following before committing:
+```bash
+citus_indent
```

Comment:
Also we should instruct the user to get back in their directory where they have the pg_auto_failover sources to run `citus_indent`, as I suppose that's where it should run. Not in the `tools/uncrustify` directory, right?